.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "workbench.colorCustomizations": {}
```

Comment:
I think it would be best to add `.vscode` to the `.gitignore` file, as this may affect settings others are using.